net.waterfox.waterfox.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: net.waterfox.waterfox
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: waterfox
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--filesystem=xdg-download:rw"
+  - "--share=network"
+  - "--socket=pulseaudio"
+  - "--device=all"
+  - "--talk-name=org.freedesktop.Notifications"
+modules:
+  - name: dbus-glib
```

Comment:
you should ues dbus-glib from the shared-modules.

But is it necessary since you are not building from source?

Reply:
Will switch to dbus-glib, and yes it is necessary unfortunately, it throws XPCOM errors without dbus-glib